src/listeners/substrate-listener/commands/genetic-testing/dna-sample-rejected/dna-sample-rejected.handler.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@nestjs/common';
+import { CommandHandler, ICommandHandler } from '@nestjs/cqrs';
+import { DnaSampleRejectedCommand } from './dna-sample-rejected.command';
+import { NotificationDto } from '../../../../../endpoints/notification/dto/notification.dto';
+import { NotificationService } from '../../../../../endpoints/notification/notification.service';
+import { DateTimeProxy } from '../../../../../common';
+
+@Injectable()
+@CommandHandler(DnaSampleRejectedCommand)
+export class DnaSampleRejectedCommandHandler
+  implements ICommandHandler<DnaSampleRejectedCommand>
+{
+  constructor(
+    private readonly notificationService: NotificationService,
+    private readonly dateTimeProxy: DateTimeProxy,
+  ) {}
+
+  async execute(command: DnaSampleRejectedCommand) {
+    const dnaSample = command.dnaSample;
+
+    const sampleRejectedNotification: NotificationDto = {
+      role: 'Customer',
+      entity_type: 'Genetic Testing Tracking',
+      entity: 'QC Failed',
+      description: `Your sample from ${dnaSample.orderId} has been rejected. See your test details.`,
```

Comment:
Your sample from ${SpecimenNumber} has been rejected. Click here to see your order details.